camera/CMakeLists.txt

```diff
@@ -1,7 +1,65 @@
 cmake_minimum_required(VERSION 3.2)
 project(camera)
+set(ORESATLIVESW_VERSION_MAJOR "0")
+set(ORESATLIVESW_VERSION_MINOR "0")
+set(ORESATLIVESW_VERSION_PATCH "0")
+set(ORESATLIVESW_VERSION_REVISION "1")
+string(CONCAT CPACK_PACKAGE_VERSION
+    "${ORESATLIVESW_VERSION_MAJOR}."
+    "${ORESATLIVESW_VERSION_MINOR}."
+    "${ORESATLIVESW_VERSION_PATCH}-"
+    "${ORESATLIVESW_VERSION_REVISION}"
+    )
+# Include and output directories
+include_directories(
+    ${PROJECT_SOURCE_DIR}
+)
+set(ORESATLIVESW_RUNTIME_OUTPUT_DIRECTORY ${PROJECT_SOURCE_DIR}/bin/${CMAKE_BUILD_TYPE})
+set(ORESATLIVESW_ARCHIVE_OUTPUT_DIRECTORY ${PROJECT_SOURCE_DIR}/lib/${CMAKE_BUILD_TYPE})
+set(ORESATLIVESW_LIBRARY_OUTPUT_DIRECTORY ${PROJECT_SOURCE_DIR}/lib/${CMAKE_BUILD_TYPE})
 
-file(GLOB sources ./*)
+# Global CPack configuration
+set(CPACK_GENERATOR "DEB")
+set(CPACK_PACKAGE_NAME "oresat-live-software-camera")
+set(CPACK_PACKAGE_CONTACT "PSAS <oresat@pdx.edu>")
+set(CPACK_DEBIAN_PACKAGE_DESCRIPTION "Oresat Live Software")
+set(CPACK_DEBIAN_PACKAGE_SECTION "net")
+set(CPACK_DEBIAN_PACKAGE_DEPENDS "libv4l-dev (>= 1.16.3-3), iw")
+set(CPACK_DEB_COMPONENT_INSTALL ON)
+set(CPACK_DEBIAN_ENABLE_COMPONENT_DEPENDS ON)
+set(CPACK_DEBIAN_PACKAGE_HOMEPAGE "https://github.com/oresat/oresat-live-software")
+
+# Determine architecture
+if(PLATFORM STREQUAL "armhf")
+    set(CPACK_DEBIAN_PACKAGE_ARCHITECTURE "armhf")
+else() # Else use host architecture
+    execute_process(COMMAND dpkg --print-architecture OUTPUT_VARIABLE CPACK_DEBIAN_PACKAGE_ARCHITECTURE OUTPUT_STRIP_TRAILING_WHITESPACE)
+    if ("${CPACK_DEBIAN_PACKAGE_ARCHITECTURE}" STREQUAL "")
+        set(CPACK_DEBIAN_PACKAGE_ARCHITECTURE "i386")
+    endif()
+endif()
 
+# RX component CPack config
+set(CPACK_DEBIAN_CAMERA_FILE_NAME "${CPACK_PACKAGE_NAME}_${CPACK_PACKAGE_VERSION}_${CPACK_DEBIAN_PACKAGE_ARCHITECTURE}.deb")
+include(CPack)
+set(CAPTURE_DESCRIPTION "OreSat Live Sofware camera capture program")
+file(GLOB sources ./*)
 add_executable(capture ${sources})
```

Comment:
I know this is existing code. I'm surprised this wouldn't this try to build `capture` from everything in the directory (including `camera.py`, `transfer.sh`, and `README.md`)... maybe `add_executable` is smart about names somehow, or ignores files that aren't compilable code. The CMake documentation doesn't say anything about that AFAICT.

Reply:
I think its probably smart enough to know what to look for but if you got some change in mind lmk